frontend/src/store/controller/selectors.ts

```diff
@@ -267,7 +267,16 @@ export const getSpO2AlarmLimitsRequest = alarmLimitsSelector(getAlarmLimitsReque
 export const getHRAlarmLimitsRequest = alarmLimitsSelector(getAlarmLimitsRequest, 'hr');
 export const getFiO2AlarmLimitsCurrent = alarmLimitsSelector(getAlarmLimitsCurrent, 'fio2');
 export const getFlowAlarmLimitsCurrent = alarmLimitsSelector(getAlarmLimitsCurrent, 'flow');
-
+const alarmLimitsRequestDraftSelector = (selector: SelectorType) =>
```

Comment:
This name is a bit too general, it implies that this function would be useful in other situations where it might not be relevant. Maybe `automaticAlarmLimitsRangeSelector` is more specific, and we can parameterize the range width (instead of hard-coding 2 into `alarmLimitsRequestDraftSelector`, we hard-code it in the function calls defining `getFiO2AlarmLimitsDraft` and `getFlowAlarmLimitsDraft`